src/azure/Model/CompositeTypeRba.cs

```diff
@@ -48,69 +48,13 @@ public override string GetBaseTypeName()
             {
                 string typeName = this.BaseModelType.Name;

```

Comment:
should we not declare this variable now, and just return   " < " + this.BaseModelType.Name; ? else return string.Empty?

Reply:
@veronicagg Yes. the 'typeName' is no longer required. I have removed with the latest commit.